go.mod

```diff
@@ -42,7 +42,7 @@ require (
 	github.com/grpc-ecosystem/grpc-gateway/v2 v2.18.1
 	github.com/hashicorp/go-metrics v0.5.2
 	github.com/noot/ring-go v0.0.0-20231019173746-6c4b33bcf03f
-	github.com/pokt-network/shannon-sdk v0.0.0-20240531140038-9a8345ef580d
+	github.com/pokt-network/shannon-sdk v0.0.0-20240603170648-a1e5bcd63e4b
```

Comment:
Make sure to squash-and merge `shannon-sdk` and update this (if needed).

Reply:
I'm paying close attention to this :)